bin/fin

```diff
@@ -1334,6 +1335,9 @@ docker_machine_create ()
 		virtualbox)
 			check_vbox_version
 
+			# Use a local copy of boot2docker.iso (in the current directory) when provisioning in offline mode
+			[[ "$offline" ]] && URL_BOOT2DOCKER="boot2docker.iso"
```

Comment:
I propose that default paths were not in current directory but in `~/.docksal`
`~/.docksal/boot2docker.iso`
`~/.docksal/images.tar`